package.json

```diff
@@ -255,9 +255,9 @@
     "@lezer/common": "1.2.1",
     "@lezer/highlight": "1.2.0",
     "@lezer/lr": "1.3.3",
-    "@locker/near-membrane-dom": "0.13.3",
-    "@locker/near-membrane-shared": "0.13.3",
-    "@locker/near-membrane-shared-dom": "0.13.3",
+    "@locker/near-membrane-dom": "0.13.6",
+    "@locker/near-membrane-shared": "0.13.6",
+    "@locker/near-membrane-shared-dom": "0.13.6",
```

Comment:
Is this necessary for the fix? Asking as it's major zero versioned and could have breaking changes in it.

Reply:
I checked all the releases and there are no breaking changes but there's a patch for a problem with chrome that we want